src/models/decisions/decision.ts

```diff
@@ -0,0 +1,17 @@
+import { EnumValue } from "../../helpers/advancedTypes";
+
+export enum VerificationDecision {
+    FALSE = "false",
+    TRUE = "true",
+    UNSURE = "unsure",
+    SKIP = "skip",
+}
+
+// A polymorphic structure that can be used to represent any decision type
+// that implements the interface
+export interface IDecision {
+    confirmed: EnumValue<VerificationDecision>;
+    identifier: symbol;
+    type: string;
```

Comment:
what is type?

Reply:
Type is no longer needed

I used to use it to identify if a decision was a `Verification` or `Classification` (Classification used to extend Verification so I could not use `instanceof`)